app/views/hyrax/batch_ingest/batches/show.html.erb

```diff
@@ -17,17 +27,18 @@
   <table class='table table-condensed'>
     <thead>
       <tr>
-        <th scope='col'>Status</th>
-        <th scope='col'>ID Within Batch</th>
-        <th scope='col'>Source Location</th>
-        <th scope='col'>Repository ID</th>
-        <th scope='col'>Error</th>
+        <th scope='col'><%= sort_link_to('Status', 'status') %></th>
+        <th scope='col'><%= sort_link_to('ID Within Batch', 'id_within_batch') %></th>
+        <th scope='col'><%= sort_link_to('Source Location', 'source_location') %></th>
+        <th scope='col'><%= sort_link_to('Repository ID', 'object_id') %></th>
+        <th scope='col'><%= sort_link_to('Error', 'error') %></th>
       </tr>
     </thead>
     <tbody>
-      <% @presenter.batch_item_presenters.each_with_index do |batch_item_presenter, index| %>
-        <%= render 'batch_item', presenter: batch_item_presenter, index: index %>
+      <% @batch_items.each_with_index do |batch_item, index| %>
```

Comment:
Same question here about using the existing presenters vs intiatiating them.

Reply:
Same answer :).